mobile-app/app/hooks/wallet/Conversion.ts

```diff
@@ -48,6 +54,15 @@ export function useConversion(props: useConversionProps): ConversionResult {
   const reservedDFI = 0.1;
 
   useEffect(() => {
+    function setAmount(transactionFee: BigNumber) {
+      setConversionAmount(
+        amount
+          .minus(type === "utxo" ? DFIUtxo.amount : DFIToken.amount)
+          .plus(type === "utxo" ? reservedDFI : 0)
+          .plus(type === "utxo" ? transactionFee : 0)
```

Comment:
converting to dfi token should have tx fee too right?

Reply:
ah yes

Reply:
but wait, if it's dfi conversion, we don't need to convert the transaction fee to utxo because transaction fee is deducting utxo not dfi